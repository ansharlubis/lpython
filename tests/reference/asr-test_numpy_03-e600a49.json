{
    "basename": "asr-test_numpy_03-e600a49",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/test_numpy_03.py",
    "infile_hash": "746b09145c989ae658a31293ea7f6f164d93bd5bdee9b62380c2e726",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_numpy_03-e600a49.stderr",
    "stderr_hash": "b96dfae487c8cfe14eda6f83f4a42efa99a114461fd38472ef67b646",
    "returncode": 2
}