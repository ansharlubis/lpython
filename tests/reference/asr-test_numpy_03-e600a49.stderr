semantic error: The symbol dtype is not found
  --> tests/../integration_tests/test_numpy_03.py:13:24
   |
13 |     newshape: i32[1] = empty(1, dtype=int)
   |                        ^^^^^^^^^^^^^^^^^^^ 
